src/guidescanpy/flask/templates/job_query.html

```diff
@@ -169,7 +169,7 @@ <h5 class="card-header">{{ region }}</h5>
                         ],
                     });
 
-                    $('.datatable tbody').on('click', 'a', function () {
+                    $('.datatable tbody').on('click', 'a[id="off_target"]', function () {
```

Comment:
pick `class` instead of `id.` I'm thinking there must be a shorter syntax for selection by class instead of changing this to `class="off_target"` (something like `a[$off_target]` or something similar).